include/dlpack/dlpack.h

```diff
@@ -208,6 +235,13 @@ typedef struct {
  *  not meant to transfer the tensor. When the borrowing framework doesn't need
  *  the tensor, it should call the deleter to notify the host that the resource
  *  is no longer needed.
+ *
+ * \note This data structure is used as Legacy DLManagedTensor
+ *       in DLPack exchange and is deprecated after DLPack v0.8
```

Comment:
@tqchen right now there is no v0.8 yet. Are you thinking what I am thinking (that to do a v0.8 release to include, e.g., `kDLBool` from #114), before making the breaking change to v1.0? If it's not what you had in mind, may I suggest to consider it? 🙂

Reply:
Thanks @leofang added here https://github.com/dmlc/dlpack/releases/tag/v0.8

Reply:
Thanks Tianqi!